docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -3,48 +3,24 @@ title: "Installing Pants"
 slug: "installation"
 hidden: false
 createdAt: "2020-02-21T17:44:53.022Z"
-updatedAt: "2022-05-03T20:48:35.453Z"
+updatedAt: "2022-07-12T00:00:00.000Z"
```

Comment:
Is this necessary?

Reply:
I believe this shows up on the docs site:

![2022-07-12 at 9 43 AM](https://user-images.githubusercontent.com/622300/178547547-f424b697-de40-4aa0-95a7-44ff7d210cfe.png)

Reply:
What happens if you omit it from the front matter? Will readme do the right thing and use the current time?

Reply:
Or we can fill this in automatically from the last git change time. We're going to need to automatically edit the front matter anyway to deal with the category problem.